From f77021d720f12767576c25d751c75cacd7478614 Mon Sep 17 00:00:00 2001
From: dcashman <dcashman@android.com>
Date: Tue, 23 Feb 2016 12:23:59 -0800
Subject: libselinux: procattr: return error on invalid pid_t
 input.

Signed-off-by: Daniel Cashman <dcashman@android.com>
---
 src/procattr.c | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/procattr.c b/src/procattr.c
index 527a0a5..c20f003 100644
--- a/src/procattr.c
+++ b/src/procattr.c
@@ -70,9 +70,9 @@ static int openattr(pid_t pid, const char *attr, int flags)
 	char *path;
 	pid_t tid;
 
-	if (pid > 0)
+	if (pid > 0) {
 		rc = asprintf(&path, "/proc/%d/attr/%s", pid, attr);
-	else {
+	} else if (pid == 0) {
 		rc = asprintf(&path, "/proc/thread-self/attr/%s", attr);
 		if (rc < 0)
 			return -1;
@@ -82,6 +82,9 @@ static int openattr(pid_t pid, const char *attr, int flags)
 		free(path);
 		tid = gettid();
 		rc = asprintf(&path, "/proc/self/task/%d/attr/%s", tid, attr);
+	} else {
+		errno = EINVAL;
+		return -1;
 	}
 	if (rc < 0)
 		return -1;
-- 
2.4.3

