From c85fd7d9c45770b31de44bb35521e2251882df10 Mon Sep 17 00:00:00 2001
From: Xin Ouyang <Xin.Ouyang@windriver.com>
Date: Tue, 9 Jun 2015 21:22:52 +0530
Subject: [PATCH] fc/sysnetwork: apply policy to network commands alternatives

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
Signed-off-by: Shrikant Bobade <Shrikant_Bobade@mentor.com>
Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/system/sysnetwork.fc | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/policy/modules/system/sysnetwork.fc b/policy/modules/system/sysnetwork.fc
index c9ec4e5ab..4ca151524 100644
--- a/policy/modules/system/sysnetwork.fc
+++ b/policy/modules/system/sysnetwork.fc
@@ -44,6 +44,7 @@ ifdef(`distro_redhat',`
 /usr/bin/dhcpcd		        --	gen_context(system_u:object_r:dhcpc_exec_t,s0)
 /usr/bin/ethtool		    --	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/bin/ifconfig		    --	gen_context(system_u:object_r:ifconfig_exec_t,s0)
+/usr/bin/ifconfig\.net-tools		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/bin/ip			        --	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/bin/ipx_configure		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/bin/ipx_interface		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
@@ -60,13 +61,16 @@ ifdef(`distro_redhat',`
 /usr/sbin/dhcpcd		--	gen_context(system_u:object_r:dhcpc_exec_t,s0)
 /usr/sbin/ethtool		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/ifconfig		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
+/usr/sbin/ifconfig\.net-tools		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/ip			--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
+/usr/sbin/ip\.iproute2			--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/ipx_configure		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/ipx_interface		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/ipx_internal_net	--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/iw			--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/iwconfig		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/mii-tool		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
+/usr/sbin/mii-tool\.net-tools		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/sbin/pump			--	gen_context(system_u:object_r:dhcpc_exec_t,s0)
 /usr/sbin/tc			--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 
-- 
2.17.1

