Fix hard coded directories in the test makefile

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

diff -ur checkpolicy-2.1.0.orig/test/Makefile checkpolicy-2.1.0/test/Makefile
--- checkpolicy-2.1.0.orig/test/Makefile	2011-07-27 14:32:54.000000000 -0500
+++ checkpolicy-2.1.0/test/Makefile	2012-01-16 18:21:32.140776138 -0600
@@ -2,8 +2,8 @@
 # Makefile for building the dispol program
 #
 PREFIX ?= $(DESTDIR)/usr
-BINDIR=$(PREFIX)/bin
-LIBDIR=$(PREFIX)/lib
+BINDIR ?= $(PREFIX)/bin
+LIBDIR ?= $(PREFIX)/lib
 INCLUDEDIR ?= $(PREFIX)/include
 
 CFLAGS ?= -g -Wall -O2 -pipe
