Subject: [PATCH] refpolicy: fix real path for cgroup

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 policy/modules/contrib/cgroup.fc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/policy/modules/contrib/cgroup.fc b/policy/modules/contrib/cgroup.fc
index b6bb46c..e214727 100644
--- a/policy/modules/contrib/cgroup.fc
+++ b/policy/modules/contrib/cgroup.fc
@@ -10,6 +10,9 @@
 /sbin/cgconfigparser		--	gen_context(system_u:object_r:cgconfig_exec_t,s0)
 /sbin/cgrulesengd		--	gen_context(system_u:object_r:cgred_exec_t,s0)
 /sbin/cgclear			--	gen_context(system_u:object_r:cgclear_exec_t,s0)
+/usr/sbin/cgconfigparser		--	gen_context(system_u:object_r:cgconfig_exec_t,s0)
+/usr/sbin/cgrulesengd		--	gen_context(system_u:object_r:cgred_exec_t,s0)
+/usr/sbin/cgclear			--	gen_context(system_u:object_r:cgclear_exec_t,s0)
 
 /var/log/cgrulesengd\.log	--	gen_context(system_u:object_r:cgred_log_t,s0)
 /var/run/cgred.*			gen_context(system_u:object_r:cgred_var_run_t,s0)
-- 
1.7.11.7

