Subject: [PATCH] fix xconsole_device_t as a dev_node.

Upstream-Status: Inappropriate [only for Poky]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 policy/modules/services/xserver.te |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/policy/modules/services/xserver.te b/policy/modules/services/xserver.te
index d40f750..5bb97e9 100644
--- a/policy/modules/services/xserver.te
+++ b/policy/modules/services/xserver.te
@@ -151,6 +151,7 @@ userdom_user_tmp_file(xauth_tmp_t)
 # this is not actually a device, its a pipe
 type xconsole_device_t;
 files_type(xconsole_device_t)
+dev_node(xconsole_device_t)
 fs_associate_tmpfs(xconsole_device_t)
 files_associate_tmp(xconsole_device_t)
 
-- 
1.7.5.4

