Subject: [PATCH] refpolicy: fix real path for nscd

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 policy/modules/contrib/nscd.fc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/contrib/nscd.fc b/policy/modules/contrib/nscd.fc
index 623b731..9e4b3d0 100644
--- a/policy/modules/contrib/nscd.fc
+++ b/policy/modules/contrib/nscd.fc
@@ -1,6 +1,7 @@
 /etc/rc\.d/init\.d/nscd	--	gen_context(system_u:object_r:nscd_initrc_exec_t,s0)
 
 /usr/sbin/nscd		--	gen_context(system_u:object_r:nscd_exec_t,s0)
+/usr/bin/nscd		--	gen_context(system_u:object_r:nscd_exec_t,s0)
 
 /var/db/nscd(/.*)?		gen_context(system_u:object_r:nscd_var_run_t,s0)
 /var/cache/nscd(/.*)?		gen_context(system_u:object_r:nscd_var_run_t,s0)
-- 
1.7.11.7

