Subject: [PATCH] fix update-alternatives for tinylogin getty

Upstream-Status: Inappropriate [only for Poky]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 policy/modules/system/getty.fc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/policy/modules/system/getty.fc b/policy/modules/system/getty.fc
index e1a1848..a0bfd2e 100644
--- a/policy/modules/system/getty.fc
+++ b/policy/modules/system/getty.fc
@@ -2,6 +2,7 @@
 /etc/mgetty(/.*)?		gen_context(system_u:object_r:getty_etc_t,s0)
 
 /sbin/.*getty		--	gen_context(system_u:object_r:getty_exec_t,s0)
+/sbin/getty\.tinylogin	--	gen_context(system_u:object_r:getty_exec_t,s0)
 
 /var/log/mgetty\.log.*	--	gen_context(system_u:object_r:getty_log_t,s0)
 /var/log/vgetty\.log\..* --	gen_context(system_u:object_r:getty_log_t,s0)
-- 
1.7.5.4

