Subject: [PATCH] fix update-alternatives for hostname

Upstream-Status: Inappropriate [only for Poky]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 policy/modules/system/hostname.fc |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/policy/modules/system/hostname.fc b/policy/modules/system/hostname.fc
index 9dfecf7..4003b6d 100644
--- a/policy/modules/system/hostname.fc
+++ b/policy/modules/system/hostname.fc
@@ -1,2 +1,3 @@
 
 /bin/hostname		--	gen_context(system_u:object_r:hostname_exec_t,s0)
+/bin/hostname\.net-tools	--	gen_context(system_u:object_r:hostname_exec_t,s0)
-- 
1.7.5.4

