Subject: [PATCH] refpolicy: fix real path for screen

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
---
 policy/modules/contrib/screen.fc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/contrib/screen.fc b/policy/modules/contrib/screen.fc
index c8254dd..4a321d1 100644
--- a/policy/modules/contrib/screen.fc
+++ b/policy/modules/contrib/screen.fc
@@ -8,6 +8,7 @@ HOME_DIR/\.screenrc		--	gen_context(system_u:object_r:screen_home_t,s0)
 # /usr
 #
 /usr/bin/screen			--	gen_context(system_u:object_r:screen_exec_t,s0)
+/usr/bin/screen-.*		--	gen_context(system_u:object_r:screen_exec_t,s0)
 
 #
 # /var
-- 
1.7.11.7

