
Upstream-Status: pending
Signed-Off-By: Xiaofeng Yan <xiaofeng.yan@windriver.com>
--- util-linux-2.21.1/libmount/src/Makefile.am	2012-03-29 21:11:14.456615759 +0800
+++ util-linux-2.21.1/libmount/src/Makefile.am.new	2012-06-21 11:02:39.971163392 +0800
@@ -30,7 +30,7 @@
 
 libmount_la_LIBADD = $(ul_libblkid_la) $(SELINUX_LIBS)
 
-libmount_la_DEPENDENCIES = $(libmount_la_LIBADD) libmount.sym libmount.h.in
+libmount_la_DEPENDENCIES = $(ul_libblkid_la) libmount.sym libmount.h.in
 
 libmount_la_LDFLAGS = -Wl,--version-script=$(ul_libmount_srcdir)/libmount.sym \
                       -version-info $(LIBMOUNT_VERSION_INFO)
